src/ts/playerutils.ts

```diff
@@ -78,15 +92,10 @@ export namespace PlayerUtils {
 
   /**
    * Detects changes of the stream type, i.e. changes of the return value of the player#isLive method.
-   * Normally, a stream cannot change its type during playback, it's either VOD or live. Due to bugs on some
-   * platforms or browsers, it can still change. It is therefore unreliable to just check #isLive and this detector
-   * should be used as a workaround instead.
    *
-   * Known cases:
+   * @deprecated use `PlayerEvent.DurationChanged` instead
```

Comment:
Please do not forget that the UI is always backwards compatible to all player releases of the major version.

Reply:
Yes I'm aware of that but I wasn't sure how to write that in a case that it should be treated as deprecated for the next major player version 🤔 

Actually just removing the `@deprecated` should be sufficient or? (When keeping the TODO below)

Reply:
I would keep all the description, add a sentence that the stream type can indeed switch since version X, make it `@deprecated since vX.Y.Z use `PlayerEvent.DurationChanged` instead` and keep the TODO

Reply:
5c5c04f